chips/nrf52/src/ficr.rs

```diff
@@ -376,6 +383,28 @@ impl Ficr {
         regs.deviceaddr0.read(DeviceAddress0::DEVICEADDRESS)
     }
 
+    pub fn full_address(&self) -> [u8; 6] {
```

Comment:
Can we just call this `address()` and remove the address function that exists? I'm not sure what the old version is for, and `make allcheck` still works after I deleted it.

Otherwise looks good.

Reply:
Sounds good. I've renamed it!

And thanks for looking over this!